campair/campair/View/ToolDictionary/ToolDictionaryMainView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ToolDictionaryMainView.swift
+//  campair
+//
+//  Created by 이가은 on 2022/06/12.
+//
+
+import SwiftUI
+
+struct ToolDictionaryMainView: View {
+    @State private var searchWord : String = ""
```

Comment:
```suggestion
    @State private var searchWord = ""
```

Int, Double, Float 가 아닌경우에는 Swift의 타입 속도을 믿고 생략해도 좋을 것 같아요 ~

Reply:
> PR 본문

Thank you young !

Reply:
didn't know this! Thank you for explan this ~ Thank you yong ><